CHANGELOG.md

```diff
@@ -1,3 +1,100 @@
+## 2023-09-14 v0.20.0
+
+> Note: IP address 0.0.0.0 is no longer advertised by a node
+
+> CLI option `store-resume-peer` has been removed
+
+## What's Changed
+
+Release highlights:
+* RLN is now part of standard release (is no longer EXPERIMENTAL feature)
+* Interop tests between nwaku and js-waku are now gating PRs and releases
+* Libwaku has been made threadsafe
```

Comment:
This is a WIP.
We had an enhancement of creating the `WakuNode` instance within the _Waku Thread_ but still pending to complete a couple more recommendations made by Jacek.

So, you can either add this suggestion or remove this line until we have all the Jacek's points added.

```suggestion
Libwaku has been made more threadsafe (1 out of 3 improvements applied.)
```